pkg/controller/atlasdeployment/atlasdeployment_controller.go

```diff
@@ -259,13 +259,19 @@ func (r *AtlasDeploymentReconciler) handleDeploymentBackupSchedule(ctx *workflow
 	}
 	apiScheduleRes.Policies = []mongodbatlas.Policy{apiPolicy}
 
-	currentSchedule, _, err := ctx.Client.CloudProviderSnapshotBackupPolicies.Delete(context.Background(), projectID, cName)
+	currentSchedule, response, err := ctx.Client.CloudProviderSnapshotBackupPolicies.Delete(context.Background(), projectID, cName)
 	if err != nil {
 		r.Log.Debugf("unable to delete current backup policy for project: %v:%v, %v", projectID, cName, err)
+		return fmt.Errorf("unable to delete current backup policy for project: %v:%v, %w", projectID, cName, err)
```

Comment:
code dup nit: print the error message into a string and then use it in both calls

Reply:
Done. Thank you!